integration/tests/cook/test_basic.py

```diff
@@ -209,10 +209,13 @@ def test_max_runtime_exceeded(self):
         # the value needs to be a little more than 2 times settings_timeout_interval_minutes to allow
         # at least two runs of the lingering task killer
         job_timeout_interval_seconds = (2 * settings_timeout_interval_minutes * 60) + 15
+        job_timeout_interval_ms = job_timeout_interval_seconds * 1000
+        max_runtime_ms = 5000
+        assert max_runtime_ms < job_timeout_interval_ms
```

Comment:
Why `assert` and not `self.assertLessThan`?

Reply:
This is an assertion about the logic in the test code rather than an assertion about the code being tested.